### PR TITLE
feat: add react-infinitegrid@3.0.0

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#282

### CODE REVIEW

File: packages/react-infinitegrid/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import './App.css';
+import { GridLayout } from './react-infinitegrid';
+
+class Item extends React.Component<{
```
Comment (mixed) [inline packages/react-infinitegrid/src/App.tsx]: What is the dependent version of react?

File: packages/react-infinitegrid/src/serviceWorker.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
+// existing tabs open on the page have been closed, since previously cached
+// resources are updated in the background.
+
+// To learn more about the benefits of this model and instructions on how to
+// opt-in, read https://bit.ly/CRA-PWA
+
+const isLocalhost = Boolean(
```
[the remark this answers is not in the document]
Comment (daybrush) [reply packages/react-infinitegrid/src/serviceWorker.ts]: @mixed 
it's basic code of create-react-app.
Probably the code that runs when you run `npm run start`.

File: packages/react-infinitegrid/src/App.tsx (hunk 1 above)
Comment (daybrush) [reply packages/react-infinitegrid/src/App.tsx]: @mixed 
Maybe react 15?

File: packages/react-infinitegrid/src/serviceWorker.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (mixed) [reply packages/react-infinitegrid/src/serviceWorker.ts]: @daybrush 
How about creating a `sample` folder and managing it? [like it ](https://github.com/naver/egjs-infinitegrid/tree/04cc1d07f36fdbeffeccf15b270fbbd7db7e7255/packages/react-layout/examples)

**Pull request merged**